noted/notes/v1/notes.proto

```diff
@@ -91,6 +91,7 @@ message UpdateNoteRequest {
 }
 
 message UpdateNoteResponse {
+    string note_id = 1;
 }
```

Comment:
I don't understand why we need to return the `note_id`. How is this an indication that the RPC was successful? The client is aware of the ID of the note he wants to modify because the client needs the ID to make the call in the first place. Besides, `UpdateNoteResponse` should return the updated representation of the object as per REST standards.
```suggestion
message UpdateNoteResponse {
    Note note = 1;
}
```